### ISSUE TITLE
make shortcuts configurable

### ISSUE THREAD

**Issue by umlaeute** [pure-data/pure-data#2353]
this has been on my to-do list for a long time (but obviously not with very high priority), but since this has multiple times popped up in the last days, i thought i'd open a separate feature request:

i think that Pd already has too many shortcuts, but for some functionality that is currently only reachable via the menu some people would like to have keyboard shortcuts.

as stated in https://github.com/pure-data/pure-data/issues/1729#issuecomment-1232201082:

> i think the only valid solution for that problem would be a to provide a means for all shortcuts to be configurable.
>
> see [this thread](https://lists.puredata.info/pipermail/pd-list/2019-11/126385.html) on the Pd-list, where the suggestion was not greeted unanimously.
>
> also note that depending on your language, a couple of short-cuts might be unavailable/impractical anyhow.

**Comment by porres** [comment]
> the suggestion was not greeted unanimously.

I agree with @Spacechild1 in that thread and I think the takeaway is: have good default shortcuts + make them configurable, so one does not discard the other.

Regular users simply stick with the default shortcuts for sure, and if one uses many computers, just stick to the regular shortcuts we've been dealing with for decades. Not to mention, one can save preferences and even import them and I believe shortcuts would be there, right?
